action.yml

```diff
@@ -1,28 +1,23 @@
-# See: https://docs.github.com/en/actions/creating-actions
-name: TODO:Action Name
+name: RubyGems Setup GPR
```

Comment:
```suggestion
name: Setup GitHub Package Registry for RubyGems
```

Reply:
Need to be a bit careful here. Sometimes the UI will use the file/repo name and sometimes it will use this name in the UI, which can be confusing if they don't match. This would change the order of the name, so the two names at first glance look different and might sort differently. Therefore `RubyGems Setup GitHub Package Registry`.

Basically trying to say, I have found it best to keep these names as just a human tidy title version of the repo name they are for to avoid confusing one self!

Reply:
Ah ok!